src/app/organizer/Actions.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { ReactNode, useCallback } from 'react';
+import _ from 'lodash';
+import { t } from 'app/i18next-t';
+import styles from './Actions.m.scss';
+import { AppIcon, lockIcon } from 'app/shell/icons';
+import DropDown, { DropDownItem } from './DropDown';
+import { itemTagList, TagInfo } from 'app/inventory/dim-item-info';
+import { DimStore } from 'app/inventory/store-types';
+
+export interface ColumnStatus {
+  id: string;
+  content: ReactNode;
+  enabled: boolean;
+  subColumnIds: string[];
+}
+
+const bulkItemTags = Array.from(itemTagList);
+bulkItemTags.push({ type: 'clear', label: 'Tags.ClearTag' });
+
+function Actions({
```

Comment:
It'd be great to have a brief comment on all the components explaining what it's for.